src/FilterLists.Api/DependencyInjection/Extensions/IWebHostExtension.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Data.SqlClient;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Polly;
```

Comment:
Also curious if adding retry logic with Polly is necessary. I've never had seeding fail in the way that it is currently called, but maybe making it async and moving it into `Main()` cause some known failures where retrying is beneficial?

Reply:
Polly is necessary if your DB server delay start(while you deploy it using as container in docker...)
migration can't fail for network related exception. The retry options for DbContext only apply to transient exceptions.!